src/app/instruction-generators/new-due-time-transformer.ts

```diff
@@ -9,7 +9,7 @@ type HasNewDueTime = {
 
 export class NewDueTimeTransformer extends TokenOptionInstructionTransformer<HasNewDueTime> {
     public get infoDescription(): string {
-        return 'Extend Assignment/Quiz Lock Date To';
+        return 'Extends Canvas Assignment/Quiz Until Date To'; // TODO: Phrasing may need more work
```

Comment:
It's quite hard to clarify distinctions between until date and due date to students. We could attempt to not use the word "Date" here (maybe something like "Allow Submit/Take Canvas Assignment/Quiz Until").

Reply:
Change is made in new commits.